backend/core/urls.py

```diff
@@ -1,4 +1,8 @@
 from django.urls import path
-from .views import index
+from rest_framework import routers
+from .views import *
 
-urlpatterns = [path("", index)]
+router = routers.DefaultRouter()
+router.register('Posts', PostsViewSet)
```

Comment:
não deveria ser com p minusculo, já que é a url pra referenciar o viewset?
```suggestion
router.register('posts', PostsViewSet)
```